tests/messages/test_pofile.py

```diff
@@ -79,7 +79,7 @@ def test_read_multiline(self):
                          message.id)
 
     def test_fuzzy_header(self):
-        buf = StringIO(r'''\
+        buf = StringIO(r'''
```

Comment:
Out of curiosity -- is the parser still able to read files that start with a newline, i.e. is this just cleanup? :)